program/tests/utils/mod.rs

```diff
@@ -126,6 +129,141 @@ pub async fn create_rule_set_on_chain_with_loc(
         context.last_blockhash,
     );
 
+    assert!(
+        create_tx.message.serialize().len() <= 1232,
+        "Transaction exceeds packet limit of 1232"
+    );
```

Comment:
It won't even make it to the IX builder, though, since the packet won't be able to be sent over the network. I think we may want to rework *all* tests to do something like this since the banks client is apparently just weird.

Reply:
Oh, nevermind, I guess it will make it, but it feels weird to add still. Just since it's only BPF tests that will have this problem.